src/webui/modules/home/index.js

```diff
@@ -116,8 +121,4 @@ export default class Home extends React.Component {
   renderLoading() {
     return <Loading text="Loading..." />;
```

Comment:
If strings are `static` each render does not create a new string Object in memory. Either constant also does the same effect